src/Submit.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Button from './Button';
+import FormGroup from './FormGroup';
+import InputBase from './InputBase';
+
+class Submit extends InputBase {
+  renderFormGroup(children) {
+    let {bsStyle, ...other} = this.props; /* eslint no-unused-vars: 0 */
```

Comment:
There was a fix against input to address #507. The fix @AlexKVal previously provided was to not pass `bsStyle` into `FormGroup`.